example_configs/tomcat.json

```diff
@@ -0,0 +1,34 @@
+{
+  "rules": [
+    {
+      "pattern": "Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>requestCount:",
+      "name": "tomcat_request_count",
```

Comment:
Yes, I have made a setup with a working prom tomcat 8 example:) I have implemented a ServletContextListener to start the jmxCollector and other java default registries.